[BUG] images make target fails with `No such file or directory: '/app-root/ols/version.py`

# Bug Description

## **Output of `OLS` version**
N/A: Build failure.

## **Describe the bug**

When one attempts to build image with `make images` the build fails due to missing `version.py` file.

## **To Reproduce**

Steps to reproduce the behavior:

1. Execute `make images`

## **Expected behavior**

Build finishes without error

## **Trace**

```
[3/3] STEP 9/16: RUN pip3.11 install --no-cache-dir .
Processing /app-root
  Installing build dependencies: started
  Installing build dependencies: finished with status 'done'
  Getting requirements to build wheel: started
  Getting requirements to build wheel: finished with status 'done'
  Preparing metadata (pyproject.toml): started
  Preparing metadata (pyproject.toml): finished with status 'error'
  error: subprocess-exited-with-error
  
  × Preparing metadata (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [24 lines of output]
      Traceback (most recent call last):
        File "/usr/lib/python3.11/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 351, in <module>
          main()
        File "/usr/lib/python3.11/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 333, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "/usr/lib/python3.11/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 152, in prepare_metadata_for_build_wheel
          return hook(metadata_directory, config_settings)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "/tmp/pip-build-env-3r1r6cq8/overlay/lib/python3.11/site-packages/pdm/backend/__init__.py", line 43, in prepare_metadata_for_build_wheel
          return builder.prepare_metadata(metadata_directory).name
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "/tmp/pip-build-env-3r1r6cq8/overlay/lib/python3.11/site-packages/pdm/backend/wheel.py", line 102, in prepare_metadata
          self.initialize(context)
        File "/tmp/pip-build-env-3r1r6cq8/overlay/lib/python3.11/site-packages/pdm/backend/base.py", line 193, in initialize
          self.call_hook("pdm_build_initialize", context)
        File "/tmp/pip-build-env-3r1r6cq8/overlay/lib/python3.11/site-packages/pdm/backend/base.py", line 151, in call_hook
          getattr(hook, hook_name)(context, *args, **kwargs)
        File "/tmp/pip-build-env-3r1r6cq8/overlay/lib/python3.11/site-packages/pdm/backend/hooks/version/__init__.py", line 60, in pdm_build_initialize
          getattr(self, f"resolve_version_from_{source}")(context, **options)
        File "/tmp/pip-build-env-3r1r6cq8/overlay/lib/python3.11/site-packages/pdm/backend/hooks/version/__init__.py", line 67, in resolve_version_from_file
          with open(version_source, encoding="utf-8") as fp:
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      FileNotFoundError: [Errno 2] No such file or directory: '/app-root/ols/version.py'
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
Error: building at STEP "RUN pip3.11 install --no-cache-dir .": while running runtime: exit status 1
make: *** [Makefile:17: images] Error 1

```

--- a/Containerfile
+++ b/Containerfile
@@ -34,9 +34,9 @@ COPY --from=lightspeed-rag-embeddings /rag/embeddings_model ./embeddings_model
 # (avoid accidental inclusion of local directories or env files or credentials)
 COPY runner.py pyproject.toml LICENSE README.md ./
 
-RUN pip3.11 install --no-cache-dir .
-
 COPY ols ./ols
+
+RUN pip3.11 install --no-cache-dir .
 
 # this directory is checked by ecosystem-cert-preflight-checks task in Konflux
 COPY LICENSE /licenses/

